src/App.tsx

```diff
@@ -46,12 +46,12 @@ function App() {
 
   return (
     <SourceTargetContextProvider connections={connections}>
-      <ApiCallsContextProvider>
-        <SubscriptionsContextProvider>
-          <MessageContextProvider>
-            <SnackbarProvider>
-              <KeyringContextProvider>
-                <AccountContextProvider>
+      <KeyringContextProvider>
```

Comment:
Is there a reason for this reorganising?

Reply:
Yes! the order matters because of the props chain through the high order components. So ApiCallsContextProvider needs to have the props ready from the KeyringContextProvider. Same with ApiCallsContextProvider -> AccountContextProvider